presto-common/src/main/java/com/facebook/presto/common/type/AbstractVarcharType.java

```diff
@@ -146,6 +147,11 @@ public void writeSlice(BlockBuilder blockBuilder, Slice value, int offset, int l
         blockBuilder.writeBytes(value, offset, length).closeEntry();
     }
 
+    public void writeBytes(AbstractVariableWidthBlockBuilder blockBuilder, byte[] bytes, int offset, int length)
+    {
+        blockBuilder.writeBytes(bytes, offset, length).closeEntry();
+    }
```

Comment:
Is this only used in tests? If yes, maybe move to test-only classes?

Reply:
This is going to be used in Jolly , Presto will not use it for now.